### DISCUSSION TITLE
Sunbird Installation issues as per given instructions "Sunbird Infra Requirements" in the Diksha tender.

### DISCUSSION THREAD

**Discussion by rajeev8080** [Sunbird-Knowlg/Community#51] [Documentation Issues]
Hi Sunbird/Diksha Team,

 As per DIC RFP for " DIKSHA Platform" , Bidder will have to install the Sunbird platform based on the given installation instructions : http://docs.sunbird.org/latest/developer-docs/server-installation/
But there are many errors encountered while installing the sunbird Platform , We need urgent help from Sunbird/Diksha Team who can sort the issues. We have attached the errors Document, You can see there are some application binaries / components which are required to run the complete sunbird application are either failing during builds or during deployment phase, again by using sunbird automation jobs. 

Thus to troubleshoot those we need the sunbird application team, as it is a complete infrastructure as a code deployment & hence we may not be able to solve the issues and challenges arising out of source code.

[sunbird-install-error-document.docx](https://github.com/Sunbird-Knowlg/Community/files/9383640/sunbird-install-error-document.docx)

**Comment by beepdot** [comment]
@rajeev8080 Can you please raise one discussion forum thread per issue here - https://github.com/orgs/Sunbird-Ed/discussions ? This forum is specific to the Knowlg building block. What you are trying to setup is the whole of Sunbird, which is Sunbird-Ed. More reading when you get some time here - https://sunbird.org/building-blocks

It would be difficult to answer all your issues in a single thread and that would also not be very useful to community who might face similar issues.

Please add all relevant information while raising a query such as
- Release version
- Github tags used
- Error logs and messages
- Images if possible
- Jenkins console logs etc

Please tag @gandham-santhosh and @keshavprasadms github ids for faster response around the forums. Looking forward to seeing your queries.

**Reply by shounakacharya** [reply]
I have also raised this in Sunbird-ED https://github.com/orgs/Sunbird-Ed/discussions/76

**Comment by shounakacharya** [comment]
Hi @keshavprasadms,

I appreciate your comment. However, since this activity is being performed for doing a sunbird setup for an RFP and considering the number of errors that we are facing, and also the timelines in which to get the demo up and running, it would be very difficult for us to go and paste them separately in separate threads. If you could please help us in resolving these here, instead of duplicating the efforts again, we would be highly obliged.
cc. @gandham-santhosh

Thanks
Shounak

**Comment by beepdot** [comment]
@AmiableAnil Please close this thread. These are being answered in other threads.
